cmd/wallet-web/src/components/Vaults/VaultCard.vue

```diff
@@ -29,7 +30,7 @@
           </span>
         </div>
       </div>
-    </div>
+    </router-link>
     <div
```

Comment:
This should be a `<button>` with `@click="showAddVault = !showAddVault"` event handler (remove it from the `<img>` tag on line 54)

Reply:
removed, fixed the bug